<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InCompListWidget</class>
 <widget class="QWidget" name="InCompListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 12pt &quot;楷体&quot;;
}
#InCompListWidget
{
background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:1 rgb(226,238,243),
                                stop:0 rgb(240,207,214));
}
QPushButton
{
font: 12pt &quot;楷体&quot;;
font-weight:bold;
border-radius:8px;
border:2px solid gray;
background-color: rgba(202, 221, 245,0.7);
height:35;
}
QPushButton:hover
{
font: 13pt &quot;楷体&quot;;
font-weight:bold;
border:2px solid;
background-color: rgb(202, 221, 245);
}</string>
  </property>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>20</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;楷体&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>网络工程2021级1班团支部</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>90</y>
     <width>280</width>
     <height>477</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:2px solid gray;
border-radius:10px;
margin-top:11px;
}
QGroupBox::title {
      subcontrol-origin: margin;
      subcontrol-position: top;
}</string>
   </property>
   <property name="title">
    <string>未完成人员名单</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCopyList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>110</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>复制名单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>590</y>
     <width>110</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
